client/src/components/ColorList.js

```diff
@@ -21,10 +21,26 @@ const ColorList = ({ colors, updateColors }) => {
     // Make a put request to save your updated color
     // think about where will you get the id from...
     // where is is saved right now?
+    axios()
+      .put(`http://localhost:5000/api/colors/${colorToEdit.id}`, colorToEdit)
+      .then(res => {
+        document.location.reload();
```

Comment:
Like we discussed in our 1:1, this a wrong way to re render a component. Instead use the setter function `(setColorList)`  in `BubblePage.js` sent as to the `colorList` component as `updateColors`  to update the state. Like this 

```
 updateColors(
          colors.map(color => {
            if (color.id === colorToEdit.id) {
              return response.data;
            }
            return color;
          })
        );
```

And then to make the edit form disappear, setEditing to false
`setEditing(false)`